src/useScrollSync.ts

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+
+interface Options {
+  mode: "proportional" | "absolute";
```

Comment:
I think it’s odd to make `Mode` a required option but `axis` not. I would suggest to make both options optional (hence the name) and use same defaults as the components do. Alternatively, if you think it’s better, let’s make both options required. WYT?

Reply:
Yes, I agree. I've made the relevant changes now. Let me know what you think.